### ISSUE TITLE
Whitespace removed when deserializing mixed content

### ISSUE THREAD

**Issue by jpd236** [pdvrieze/xmlutil#84]
Using the test classes in TestMixedTypesafe.kt, we can serialize the following:

```kotlin
TypedMixed {
    text("a")
    elem(TypedMixed.E {
        text(" ")
    })
    text("b")
}
```

to `<mixed>a<e> </e>b</mixed>`. But if we then deserialize the same input, the whitespace is lost:

```
expected: <TypedMixed([Text('a'), Object(E([Text(' ')])), Text('b')])>
but was: <TypedMixed([Text('a'), Object(E([])), Text('b')])>
```

Is there any way to preserve this whitespace, either in a specific tag, or (less preferably) altogether when parsing?

**Comment by pdvrieze** [comment]
I've changed the code to not drop the whitespace in this (and some other) contexts. It is now in dev (and a snapshot should be in progress).

**Comment by pdvrieze** [comment]
Now released in 0.84.1

**Comment by jpd236** [comment]
Verified on my side - fixes the issue I was having and doesn't appear to introduce any immediate/obvious unexpected regressions. Thanks!
